packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -80,12 +87,29 @@ export class CarequalityManagementAPIImpl implements CarequalityManagementAPI {
       default:
         throw new Error("API mode not supported.");
     }
+    this.maxBackoff = options?.maxBackoff
+      ? dayjs.duration(options.maxBackoff, "seconds").asMilliseconds()
+      : DEFAULT_MAXIMUM_BACKOFF.asMilliseconds();
 
     this.api = axios.create({
       timeout: options?.timeout ?? DEFAULT_AXIOS_TIMEOUT.asMilliseconds(),
       baseURL: baseUrl,
       httpsAgent: this.httpsAgent,
     });
+
+    axiosRetry(this.api, {
+      retries: options?.retries ?? DEFAULT_MAX_RETRIES,
+      retryDelay: retryCount => {
+        const exponentialDelay = Math.pow(2, retryCount);
+        const jitter = Math.random();
+        const delayWithJitter = (exponentialDelay + jitter) * BASE_DELAY.asMilliseconds();
+        return Math.min(delayWithJitter, this.maxBackoff);
+      },
+      retryCondition: error => {
+        return error.response?.status !== 200;
```

Comment:
Don't we only want to retry on error? 2xx is success, 3xx is redirection - [reference](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes).

Also, do we want to retry on 4xx? It would be useful to have a link to the CQ spec indicating this behavior, maybe on the PR description or on Notion and a link from here to it?